FizzBuzz - single parameter.rb

```diff
@@ -0,0 +1,50 @@
+def fizz_buzz
+
+  puts "Pick a number:"
+  is_fb = gets.chomp
+  is_fb = is_fb.gsub(/,/, '.')
+
+  def print_fizz_buzz(is_fb)
+    if
+      is_fb%3 == 0 && is_fb%5 == 0
+      puts "FizzBuzz"
+    elsif is_fb%3 == 0
+      puts "Fizz"
+    elsif is_fb%5 == 0
+      puts "Buzz"
+    else
+      puts "No FizzBuzz for you, sport!"
+    end
+  end
+
+  begin
```

Comment:
nice `begin-rescue`! Please try to rescue a specific class of exception. Rescuing without error class specified will result in catching **all** errors, even those you did not intend to catch!(typos for example)

Reply:
I've read that rescuing all the exceptions is rather wrong, all of the errors on the other hand - is ok.  That's why I used the default `StandardError`. But I can see how it may be redundant in this particular case.